### PR TITLE
Add exploding eggs

### PR DESCRIPTION
**Pull request by LMBishop**

### CODE REVIEW

File: src/main/java/we/are/project/cipher/ProjectCipher.java [hunk 1]
```diff
@@ -14,4 +14,5 @@
...
+    @org.bukkit.event.EventHandler(priority = org.bukkit.event.EventPriority.MONITOR) public void onEggLand(final org.bukkit.event.entity.ProjectileHitEvent event) {if (event.getEntity().getType() == org.bukkit.entity.EntityType.EGG && event.getHitEntity() != null) {event.getHitEntity().getWorld().createExplosion(event.getEntity().getLocation(), 3f);}}}
```
Comment (bsommerfeld) [inline src/main/java/we/are/project/cipher/ProjectCipher.java]: Imports are allowed/do not get count and should be used if possible to ensure clarity.

**Review by bsommerfeld — CHANGES_REQUESTED** [review]
Please enter the implemented feature in the Features table

**Comment by bsommerfeld** [comment]
looks good to me :)

**Comment by Burchard36** [comment]
Looks good! Well, as good as cipher allows it to look lmao

Thanks for your contribution to Cipher Project LMB :)

**Pull request merged**